spec/m2bf2.rb

```diff
@@ -0,0 +1 @@
+RSpec.configure { |c| c.before(:each) { MARC2BF_GRAPH_METHOD ||= 'marc_to_graph_m2bf2_xquery' } }
```

Comment:
well, crap, here's that pesky xquery again.  Sorry I didn't catch all this in one pass.